src/java/org/apache/cassandra/auth/MutualTlsAuthenticator.java

```diff
@@ -87,6 +101,18 @@ public MutualTlsAuthenticator(Map<String, String> parameters)
         }
         certificateValidator = ParameterizedClass.newInstance(new ParameterizedClass(certificateValidatorClassName),
                                                               Arrays.asList("", AuthConfig.class.getPackage().getName()));
+
+        String maxCertificateAgeConfig = parameters.get(MAX_CERTIFICATE_AGE_CONFIG);
```

Comment:
One thing i'm wondering is whether the parameter should be on the authenticator or in the validator.

I noticed that the certificate age itself is evaluated in `SpiffeCertificateValidator`.  Since we use `ParameterizedClass` on `validator_class_name`, technically we should be able to pass parameters to it.

If you pass the parameter to the validator, then you don't need logic in both `MutualTlsAuthenticator` and `MutualTlsInternodeAuthenticator` for parsing out the certificate age.

Reply:
On the other hand there's an argument to move the evaluation out of the validator, so all `MutualTlsCertificateValidator` impleemntations don't have to implement logic to validate the certificate age.

Reply:
yeah good point, I think there's more value in moving the validation out of the validator.